pkg/multitenant/mtinfopb/info.proto

```diff
@@ -58,6 +63,9 @@ message TenantInfo {
   optional cockroach.multitenant.tenantcapabilitiespb.TenantCapabilities capabilities = 6 [
     (gogoproto.nullable) = false
   ];
+
+  // available tag: 3.
```

Comment:
mentioned above but I think we typically don't reuse an ID and instead `reserved x;` on the line where it was to make the removal/addition pairing obvious in the diff.